nsnet2/nsnet2.js

```diff
@@ -50,15 +50,21 @@ export class NSNet2 {
     });
     const [gru94, gru93] = this.builder_.gru(transpose31, weight192, recurrentWeight193, frames, this.hiddenSize,
         {bias: bias194, recurrentBias: recurrentBias194, initialHiddenState: initialState92, returnSequence: true});
-    const squeeze95 = this.builder_.squeeze(gru93, {axes: [1]});
+    // Use reshape to implement squeeze(gru93, {axes: [1]});
+    const squeezed95_shape = gru93.shape();
```

Comment:
Be consistent with the following one that is `squeeze158_shape`
```suggestion
    const squeeze95_shape = gru93.shape();
```

Reply:
Thanks, fixed.